js/script.js

```diff
@@ -0,0 +1,7 @@
+const firstNumber = Number(prompt("Enter the first number:"));
+const secondNumber = Number(prompt("Enter the second number:"));
+const thirdNumber = Number(prompt("Enter the third number:"));
+
+const average = (firstNumber + secondNumber + thirdNumber) / 3;
```

Comment:
On a real project, don't forget about some validation
A user can input letters instead of numbers, then the program output will be incorrect

In your solution will be enough to add if and check whether entered values are not NaN